ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/SteamP2PClient.cs

```diff
@@ -354,6 +271,157 @@ private void PacketSequenceComplete(BMSByte data, int groupId, byte receivers, b
 				FireRead(frame, Server);
 		}
 
+		private void ReceiveNetworkData()
+		{
+			try
+			{
+				ReadPacket();
+			}
+			catch (SocketException ex)
+			{
+				// This is a common exception when we exit the blocking call
+				Logging.BMSLog.LogException(ex);
+				Disconnect(true);
+			}
+
+			// Check to see if the headers have been exchanged
+			if (!initialConnectHeaderExchanged)
```

Comment:
Is this block a stub or should it have the validation stuff from the previous code?